Dockerfile

```diff
@@ -1,36 +1,25 @@
-FROM debian:7.8
-MAINTAINER Mike Babineau michael.babineau@gmail.com
+# As of March 18, 2020 - corona virus scare time
 
-ENV \
-    ZK_RELEASE="http://www.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz" \
-    EXHIBITOR_POM="https://raw.githubusercontent.com/Netflix/exhibitor/d911a16d704bbe790d84bbacc655ef050c1f5806/exhibitor-standalone/src/main/resources/buildscripts/standalone/maven/pom.xml" \
-    # Append "+" to ensure the package doesn't get purged
-    BUILD_DEPS="curl maven openjdk-7-jdk+" \
-    DEBIAN_FRONTEND="noninteractive"
+# https://www.debian.org/releases/
+FROM debian:stretch
 
-# Use one step so we can remove intermediate dependencies and minimize size
-RUN \
-    # Install dependencies
-    apt-get update \
-    && apt-get install -y --allow-unauthenticated --no-install-recommends $BUILD_DEPS \
+MAINTAINER Mike Babineau michael.babineau@gmail.com
 
-    # Default DNS cache TTL is -1. DNS records, like, change, man.
-    && grep '^networkaddress.cache.ttl=' /etc/java-7-openjdk/security/java.security || echo 'networkaddress.cache.ttl=60' >> /etc/java-7-openjdk/security/java.security \
+ENV DEBIAN_FRONTEND="noninteractive"
 
-    # Install ZK
-    && curl -Lo /tmp/zookeeper.tgz $ZK_RELEASE \
+# Ref: https://stackoverflow.com/questions/57031649/how-to-install-openjdk-8-jdk-on-debian-10-buster
+RUN apt-get update -y && apt-get install curl maven wget gnupg software-properties-common openjdk-8-jdk-headless openjdk-8-jdk -y \
+    && curl -Lo /tmp/zookeeper.tar.gz "https://downloads.apache.org/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz" \
     && mkdir -p /opt/zookeeper/transactions /opt/zookeeper/snapshots \
-    && tar -xzf /tmp/zookeeper.tgz -C /opt/zookeeper --strip=1 \
-    && rm /tmp/zookeeper.tgz \
-
+    && tar -xvf /tmp/zookeeper.tar.gz -C /opt/zookeeper --strip=1 \
+    && rm /tmp/zookeeper.tar.gz \
     # Install Exhibitor
     && mkdir -p /opt/exhibitor \
-    && curl -Lo /opt/exhibitor/pom.xml $EXHIBITOR_POM \
+    && curl -Lo /opt/exhibitor/pom.xml "https://raw.githubusercontent.com/Netflix/exhibitor/master/exhibitor-standalone/src/main/resources/buildscripts/standalone/maven/pom.xml" \
```

Comment:
We might want to host this config file ourselves since executor is not under netflix anymore and we don't know when they might remove the links.